pkg/cloudprovider/aws/fleet/capacity.go

```diff
@@ -99,3 +100,22 @@ func (c *Capacity) GetTopologyDomains(ctx context.Context, key cloudprovider.Top
 		return nil, fmt.Errorf("unrecognized topology key %s", key)
 	}
 }
+
+func (c *Capacity) Delete(ctx context.Context, nodes []*v1.Node) ([]*string, error) {
```

Comment:
Since the EC2 API returns this information, I wanted to keep it, in case we do need it in the future. It's only used for logging here, but I think we may end up using the ids in the future. Happy to talk about this if you think I should get rid of it.